GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
```

Comment:
Given the removal of line 21, what would "active members" entail? May want to simply strike the word "active".